package.json

```diff
@@ -12,22 +12,34 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "lerna --scope '@magento/{pwa-buildpack,peregrine}' exec -- npm run build",
-    "clean": "lerna --scope '@magento/{pwa-buildpack,peregrine}' exec -- npm run clean",
-    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "danger": "danger-ci",
+    "reset": "lerna clean --yes && rm -rf ./node_modules && npm install",
+    "build": "lerna --stream --scope '@magento/{pwa-buildpack,peregrine}' exec -- npm run build",
     "postinstall": "lerna bootstrap && npm run build",
+    "watch:buildpack": "cd packages/pwa-buildpack && npm run -s watch",
+    "watch:peregrine": "cd packages/peregrine && npm run -s watch",
+    "watch:dev-server": "nodemon --exec npm --cwd packages/venia-concept start",
+    "watch:venia": "concurrently -c yellow.dim,green.dim,cyan.dim -n ␢,℗,☄︎  'npm run watch:buildpack' 'npm run watch:peregrine' 'npm run watch:dev-server'",
```

Comment:
Is this really the plan? Watch all of these at the same time? I'd rather just watch `venia`, personally. Can we make this one `watch:all` or something?

Reply:
This is definitely what you want. This has the exact same functionality that `npm start` in venia does today, with three key improvements:
1. If you change a Peregrine source file (e.g. moving a concept from Venia to Peregrine) it will hot reload and stay up to date, instead of requiring a manual Peregrine rebuild
2. If you change a Buildpack source file (e.g. changing the behavior of a plugin or DevServer), it will shut down the dev server, rebuild Buildpack, and restart the dev server to reflect the new changes.
3. You can run it from project root.

If you don't touch Peregrine or Buildpack, it's functionally the same.

Reply:
If you don't touch `peregrine` or `buildpack`, is it more expensive to watch them? Does it bog your system down more? That's my only concern.

Reply:
Yes, I think that would help. 👍